lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -25,6 +25,7 @@ use Try::Tiny;
 use DBIx::Class::Timestamps 'now';
 use Mojo::Asset::Memory;
 use Mojo::File 'path';
+use OpenQA::Setting;
```

Comment:
Why not adding this already after the other `OpenQA::` uses? It makes no difference but would look nicer :-)